code/game/objects/items/weapons/RCD.dm

```diff
@@ -11,312 +11,180 @@ RCD
 	icon_state = "rcd"
 	opacity = 0
 	density = 0
-	anchored = 0
+	anchored = 0.0
 	flags = CONDUCT
-	force = 10
-	stamina_percentage = 0.5
-	throwforce = 10
+	force = 10.0
+	throwforce = 10.0
 	throw_speed = 3
 	throw_range = 5
 	w_class = 3
 	materials = list(MAT_METAL=100000)
 	origin_tech = "engineering=4;materials=2"
-	req_access_txt = "11"
-	var/datum/effect_system/spark_spread/spark_system
+	var/max_matter = 100
```

Comment:
This is a change in the max matter. You've reduced it from 160. Is this intentional?